apis/r/tools/get_tarball.R

```diff
@@ -1,26 +1,21 @@
 #!/usr/bin/env Rscript
 
-## version pinning info
-tiledb_core_version <- "2.19.1"
-# 8-nybble hash for 2.19.0 and 2.19.1 only. Please see https://github.com/TileDB-Inc/TileDB/pull/4599.
-tiledb_core_sha1 <- "29ceb3e7"
+# Please use scripts/update-tiledb-version.py in this repo's base directory to
+# update URLs in this file as well as the FindTileDB_EP.cmake file within the
+# libtiledbsoma directory.
 
 if ( ! dir.exists("inst/") ) {
     stop("No 'inst/' directory. Exiting.", call. = FALSE)
 }
 
-makeUrl <- function(arch, ver=tiledb_core_version, sha1=tiledb_core_sha1) {
-    sprintf("https://github.com/TileDB-Inc/TileDB/releases/download/%s/tiledb-%s-%s-%s.tar.gz", ver, arch, ver, sha1)
-}
-
 isMac <- Sys.info()["sysname"] == "Darwin"
 isLinux <- Sys.info()["sysname"] == "Linux"
 
 if (isMac) {
     arch <- system('uname -m', intern = TRUE)
-    url <- makeUrl(paste0("macos-", arch))
+    url <- "https://github.com/TileDB-Inc/TileDB/releases/download/2.19.1/tiledb-macos-x86_64-2.19.1-29ceb3e7.tar.gz"
```

Comment:
This compute the macOS architecture, stores it in `arch` and then ignores it.  So this breaks installation on non-x86_64 macOS machine based on the arm64 architecture (ie M1, M2, M3).

Reply:
Thanks @eddelbuettel !